lesson_3/ex1_solution.sh

```diff
@@ -1 +1,15 @@
-docker pull
+docker pull postgres:latest
+docker image ls postgres
+docker run -it -e POSTGRES_HOST_AUTH_METHOD=trust  --entrypoint /bin/bash postgres
```

Comment:
o entrypoint aquí non impide lanzar un comando así que podes entrar con /bin/bash (non se lanzará o postgres pero non nos importa), ou lanzar e entrar despois con docker exec -it container_name /bin/bash